packages/wonder-stuff-ci/src/mobile-release-utils.ts

```diff
@@ -0,0 +1,70 @@
+import execProm from "./exec-prom";
+
+/**
+ *
+ * @returns all release tags sorted creation time ascending
+ */
+export const getMobileReleaseTags = async (): Promise<Array<string>> => {
+    const tags = await allMobileReleaseTags();
+    return tags
+        .filter((tag) =>
+            tag.match(/^(ios|android|unified)-(\d+\.\d+\.\d+(-\w*)*)$/i),
+        )
+        .sort(compareVersions);
+};
+
+/**
+ * Compares two versions of the form <num>.<num>.<num>
+ * @param v1
+ * @param v2
+ * @returns int, 1 if v1 > v2, -1 if v1 < v2, 0 if v1 == v2
+ */
+export const compareVersions = (v1: string, v2: string) => {
+    const v1v = v1.includes("-") ? v1.split("-")[1] : v1;
+    const v2v = v2.includes("-") ? v2.split("-")[1] : v2;
```

Comment:
**question:** why are we checking for `"-"` here?  The docstring says the version should be of the form `<num>.<num>.<num>`.

Reply:
Ah, I need to update the docstring! tags are in the form `unified-7.9.0`, but this function also takes in raw version string `7.9.0`